code_examples/core_features/getting_started/06_fetch_endpoint_data.ts

```diff
@@ -1,14 +1,22 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import axios from 'axios'
 
 import * as Kilt from '@kiltprotocol/sdk-js'
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
+// TODO: Use exported type when https://github.com/KILTprotocol/sdk-js/pull/631 is merged.
+
 export async function main(
   endpoints: Kilt.DidServiceEndpoint[]
 ): Promise<Kilt.IRequestForAttestation> {
-  const endpointRequestData = await axios
-    .get<Kilt.IRequestForAttestation>(endpoints[0].urls[0])
-    .then((response) => response.data)
+  // Define the type of a published credential
+  type PublishedCredential = {
+    credential: Kilt.IRequestForAttestation
+    // Not relevant for this case, but production applications should parse this field as well
+    metadata?: any
+  }
+  const {
+    data: [{ credential }]
+  } = await axios.get<PublishedCredential[]>(endpoints[1].urls[0])
```

Comment:
Are you intentionally querying the 2nd endpoint or did you mean to query the first one?

```suggestion
  } = await axios.get<PublishedCredential[]>(endpoints[0].urls[0])
```

Reply:
Yes, Timo suggested to maintain the old endpoint, so that w3n.id would not break, and add a second endpoint with the new format.

Reply:
The idea would be to remove the old one once at least w3n.id and DIDSign have bene updated. Which we can push for with the new SDK release.